lib/krikri/index_querier.rb

```diff
@@ -0,0 +1,17 @@
+module Krikri
+  # Queries the Search Index
+  class IndexQuerier
```

Comment:
After a conversation with Tom, I am refactoring this class and renaming it `SolrResponseBuilder`.